### PR TITLE
deleted readme.  don't README

### PR DESCRIPTION
**Pull request by krstrobe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +0,0 @@
-# hello-world
-
-I’m learnin’ ’ere; ba’off!
```
Comment (kirkstrobeck) [inline README.md]: did u mean to delete this?

Comment (krstrobe) [reply README.md]: dude.  you told me to...

**Comment by krstrobe** [comment]
testing reply from email...

**Comment by kirkstrobeck** [comment]
It came thru!

bitmoji

**Pull request merged**